### Natural sequence variation of sarbecovirus Spike

We assembled an alignment of coronavirus Spike proteins from the sarbecovirus clade, which includes SARS-CoV, SARS-CoV-2, and related viruses identified in bats and other host species. The full alignment we used can be found [HERE](https://github.com/dms-view/SARS-CoV-2/blob/master/data/Spike/BloomLab2020/results/prot_map_rbd_alignment.fa). Our sequence selections include:
* The set of 30 sequences from [Letko, Marzi, and Munster (2020)](https://www.nature.com/articles/s41564-020-0688-y), which contained all known unique RBD sequences at the time of its publication. See [Extended Data Fig 1](https://www.nature.com/articles/s41564-020-0688-y/figures/6) for more information. 
* RaTG13 and RmYN02, two newly described bat CoV sequences that are the most closely related strains currently known to SARS-CoV-2.
* Two recent pangolin CoV sequences from [Lam _et al._ (2020)](https://www.nature.com/articles/s41586-020-2169-0), including the infectious virus isolated from a pangolin seized in Guanxi, and the consensus sequence from two closely related viruses assembled from metagenomic sequencing samples from pangolins seized in Guangdong.

Here, we display sequence variability from this set of natural sequences on the structure [6vyb.pdb](https://www.rcsb.org/structure/6VYB), which shows the Spike trimer. Positions are currently illustrated on all three chains simultaneously, including chain B, which contains the S<sup>B</sup> receptor binding domain (RBD) in the "up" conformation, and chains A and C which have the S<sup>B</sup> RBD in the "down" conformation.

At the mutation level, we show the frequency of each amino acid in this alignment.
At the site level, we show the entropy and number of effective amino acids for each site based on the amino-acid frequencies from this alignment.